CMSgov/design-system
# [WNMGDS-1354] Sketch Plugin to Import Tokens

## Summary

New JSON exporter added
Initial Theme definitions for Healthcare and Medicare added
Sketch token exporting plugin added

## How to test

Run `yarn install` in `/packages/design-system-tokens/`
run `yarn build tokens json`
run `npm install` in `/packages/design-system-tokens/sketch/token-exporter/`
open sketch, create new file
run `npm run start` in `/packages/design-system-tokens/sketch/token-exporter/`
select the `/packages/design-system-tokens/dist/cmsds.tokens.json` file just created
select the 4 squares icon in sketch, see screenshot
select the show color variables icon in sketch, see swatches of tokens!

![image](https://user-images.githubusercontent.com/783820/155031378-c07190ca-32f4-426f-84ed-aa85a86c9922.png)

Layer Styles (component level theme mapping) and Font's are next sprint and beyond!
